Source/Core/InputCommon/ControlReference/ExpressionParser.cpp

```diff
@@ -366,6 +399,80 @@ class VariableExpression : public Expression
   ControlState* m_value_ptr{};
 };
 
+class HotkeyExpression : public Expression
+{
+public:
+  HotkeyExpression(std::vector<std::unique_ptr<ControlExpression>> _inputs)
+      : m_inputs(std::move(_inputs))
+  {
+  }
+
+  ControlState GetValue() const override
+  {
+    if (m_inputs.empty())
+      return 0;
+
+    const bool modifiers_pressed = std::all_of(m_inputs.begin(), std::prev(m_inputs.end()),
+                                               [](const std::unique_ptr<ControlExpression>& input) {
+                                                 // TODO: kill magic number.
```

Comment:
Was this intended to be resolved in this set of changes or a future one?

Reply:
Yeah. Another PR moves the relevant constant to a header file. Didn't feel like dealing with merge conflicts.